app/Http/Controllers/ClientPortal/InvoiceController.php

```diff
@@ -46,11 +46,8 @@ class InvoiceController extends Controller
      */
     public function index(InvoiceFilters $filters)
     {
-        $client_id = auth('contact')->user()->client->id;
+        $invoices = Invoice::filter($filters)->get(); // @for-now
 
-        $invoices = Invoice::where('client_id', $client_id)->paginate(10);
```

Comment:
When we reiterate here, we can push the ->paginate() into the query builder in InvoiceFilters...

Reply:
This is for now, next update is ajax.